gulp.action.sh

```diff
@@ -30,4 +35,17 @@ for i in "$@"; do
     esac
 done
 
-npx gulp "${TASK}" --platform="${PLATFORM}" --buildMode="${BUILD_MODE}"
\ No newline at end of file
+# if [[ "${ANDROID_KEY_STORE_PASSWORD}" != "" && "${BUILD_MODE}" == "debug" || "${PLATFORM}" != "android" ]]; then
```

Comment:
Should this live here?

Reply:
I'd just check that if it's android and release, then you must have the keystore.